universal-login-react/src/ui/UFlow/Devices/ConnectedDevicesItem.tsx

```diff
@@ -31,6 +31,24 @@ export const ConnectedDevicesItem = ({devicesAmount, deviceInfo, publicKey, depl
     }
   };
 
+  const isCurrentDevice = () => deployedWallet.publicKey === publicKey;
```

Comment:
this function is used only inside `renderTrashButton`, move it inside of it and maybe turn into a variable

Reply:
Do it in a follow-up PR? This PR is getting old

Reply:
Done